launch.go

```diff
@@ -78,13 +85,30 @@ func postLaunchHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func quickLauncherHandler(w http.ResponseWriter, r *http.Request) () {
+	hostURL := settings.Get("SURVEY_RUNNER_URL")
+	surveyURL := r.URL.Query().Get("url")
+	token, err := authentication.GenerateTokenFromDefaults(surveyURL)
+
+	if err != nil {
+		http.Error(w, fmt.Sprintf("err: %v", err), 500)
```

Comment:
The error message should be `GenerateTokenFromDefaults failed err:` rather than just `err:`